bin/pycbc_multi_inspiral

```diff
@@ -617,7 +621,7 @@ with ctx:
             if len(coinc_idx) != 0:
                 coinc_idx_det_frame = {ifo : coinc_idx + time_delay_idx[ifo]
```

Comment:
Perhaps compute this dict earlier, as we do use it a bunch in what appears above?